profile/README.md

```diff
@@ -1,17 +1,28 @@
 # Project-Origin
+<!--
+The mission of the project in as a concise format as possible. It should answer what the project's purpose is and enable contributors to make a good first guess whether a suggested feature will likely be in scope for the project, or not.
+-->
 
-Project-Origin aims to create solutions to enable publicly verifiable proofs for tracking of Energy.
+Project-Origin is an open source collaboration that aims to create a public permissioned distributed ledger where everyone can validate the origin of their electricity. By being able to do so, we can support and accelerate the green transition. The long-term goal is to expand the scope from electricity, to include other utilities, because we find that the functionality can create value in a broader scope. By creating an open source platform for collaboration, we can pool ressources and accelerate the development of critical digital infrastructure that is secure and the data verifiable. 
 
-## Registry
+The open source platform is set up with decision- and work structures, to achieve effective asynchronous collaboration through transparent, accessible software, and by using decision-making processes driven by \emph{doing}.
 
-The goal of [this project](https://github.com/orgs/project-origin/projects/1) is to create the first version of a **Federated Publicly Verifiable Registry**
+## Ongoing Implementation Efforts 
+There is currently an implementation effort working on creating the first version of a federated, publicly verifiable registry. Go to the [Registry repository](https://github.com/project-origin/registry) to learn more about the implementation details and project status. 
```

Comment:
If we have only one repo so far, the pointer to the registry repository should suffice. 
Otherwise having sub headers or bullet points pointing to some of individual repositories (as done by @MartinSchmidt) is a nice idea for easier overview I think.

Reply:
@lenucksi I agree - Do you suggest making any changes to this paragraph, or should we leave it as-is for now?

Reply:
@lauranolling I think we can leave it for now and can extend things once there's more.